### PR TITLE
Getkw example running parselglossy

### PR DESCRIPTION
**Pull request by stigrj**
I put together a simple program to compute pi in two different ways, which can be chosen through an input file. Granted, the input file is not very exciting, but the actual code example can be made more interesting later.

- Changed the CMake structure to an installed superbuild from [here](https://github.com/dev-cafe/cmake-cookbook/tree/master/chapter-10/recipe-04/cxx-example)
- Added a C++ code example `pi.cpp` with corresponding target `pi.x` accepting a JSON input
- Added a Python parser script `pi.in` with corresponding target `pi` which takes a getkw input, translates to JSON and runs `pi.x`
- Added a `template.yml` defining the input
- Added an example input file in `examples/pi.inp`

The structure is very colored by the current setup in mrchem.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -5,4 +5,4 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-parselglossy = "*"
+parselglossy = {git='https://github.com/dev-cafe/parselglossy.git'}
```
Comment (robertodr) [inline Pipfile:8]: Let's not forget to pin to the PyPI version. I'll do it before my talk at ACS.

File: src/pi.cpp [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+#include "nlohmann/json.hpp"
+
+using json = nlohmann::json;
+
+int fetch_input(int argc, char **argv, json &input);
+void integration(const json &input, json &output);
+void throw_darts(const json &input, json &output);
+
+int main(int argc, char **argv) {
```
Comment (robertodr) [inline src/pi.cpp]: Can you use `EXIT_SUCCESS` and `EXIT_FAILURE` rather than `0` and `1`?

**Review by robertodr — APPROVED** [review]

**Comment by stigrj** [comment]
Hmm, the source tree has changed significantly since I cloned this :thinking:

**Comment by robertodr** [comment]
Yeah, there's two folders now: one for superbuild with `ExternalProject_Add` and one with `FetchContent`

**Comment by stigrj** [comment]
Moved everything into `externalproject`, `fetchcontent` is still the same.

**Review by robertodr — APPROVED** [review]

**Comment by robertodr** [comment]
Thanks!

**Pull request merged**